### ISSUE TITLE
lens monitoring

### ISSUE THREAD

**Issue by jbluemlein** [drewgreenwell/Falcon2Universal#1]
Could these be the lens motoring feature?
'腐': { b: 'BB 47', min: -1, max: -1, en: 'decay / rot / corrupt (fire?)' },
'廉': { b: 'B7 47', min: -1, max: -1, en: 'inexpensive / honest < google says this is honest and clean
and possibly this one for the precision mode
'孱': { b: 'B4 47', min: -1, max: -1, en: 'weak / delicate

**Comment by drewgreenwell** [comment]
Hi! I think you may be on to something but I dont remember seeing much change in traffic swapping to precise mode. Weak delicate seemed to correspond with air but I will hook up the debugger and run some tests to see if I can find a pattern there. Thanks for looking at it!

Sorry for the delay, my laser was out of commission most of last year. Back in action officially now though!

**Comment by drewgreenwell** [comment]
I did not find any correlation between those messages and precise mode. It appears to be handled entirely on the laser module and not monitored by the controller.

I didn't have any lens issues during this time but will continue to monitor.

**Comment by jbluemlein** [comment]
That seems like a thing the controller should know and be able to set.
Chinese designs right?  Like so close to being a finished product.

Are the power lines in your module soldered to nonstandard locations?  One
hot and ground in mine are soldered straight to  a regularor(transtor?) And
a random ground pad instead of through holes.

**Comment by drewgreenwell** [comment]
Yeah, I think for normal/precise it was a case of shortest path to market to get the larger modules out the door without any hardware changes for the existing machines.

I haven't opened it up recently but dont recall any non standard wiring. There might be a regulator for the fans or the usb port that i dont recall. The microcontroller in the laser has a dedicated 5V line. The 22W module doesnt use one pair of the 24V lines.

I'm changing my x axis over to a lead screw setup today. I'll open my laser to clean it and get a better look before i fire it up.

**Comment by jbluemlein** [comment]
Cool,  I have the 40w but do not have the original laser cutter it is
supposed to be used on to read the bus.  Here are the odds power
connections. Purple and brown

[image: 20250619_143116.jpg]
[image: 20250619_143127.jpg]
